### PR TITLE
Lesson22-gozde

### PR DESCRIPTION
**Pull request by 1gozde1**

### CODE REVIEW

File: Lesson22/gozde/app.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const list = document.getElementById('toDoList');
+
+formElement.addEventListener('submit', function (event) {
+	// By default, the page is reloded when form is submitted. To avoid it, we need to add `preventDefault()`;
+	event.preventDefault();
+
+  // TODO: check that input value has correct format (e.g. minimum 3 chars, no extra spaces)
+    const inputValue = formInput.value.trim();
+	if (inputValue.length < 3) {
+        alert('To-Do item must be at least 3 characters long');
```
Comment (annasvst) [inline Lesson22/gozde/app.js:12]: In the future, do not use alert function, it's considered a bad user experience. Instead, when you want to show an error message to the user, do it like we did in the form validation exercise.

**Review by annasvst — APPROVED** [review]
Approved

File: Lesson22/gozde/app.js (hunk 1 above)
Comment (1gozde1) [reply Lesson22/gozde/app.js:12]: i am asking you to be sure, what i have to do is creating a new <div> in html file (<div id="error-message" class="error"></div>) like this and correct the css and javascript files according to this new created html code?

Comment (1gozde1) [reply Lesson22/gozde/app.js:12]: <div id="error-message" class="error"></div>

Comment (1gozde1) [reply Lesson22/gozde/app.js:12]: ` <div id="error-message" class="error"></div>`

**Pull request merged**